router/session.py

```diff
@@ -0,0 +1,103 @@
+from fastapi import APIRouter, HTTPException
+import requests
+from settings import settings
+from datetime import datetime
+
+router = APIRouter(prefix="/session", tags=["Session"])
+session_db_url = settings.db_url + "session/"
+session_start_buffer_time = 900000
```

Comment:
what is this?

Reply:
Instead of using the exact start time stated by the session, we open the session 15 mins early.

Reply:
That was requested by the ops team. To let students in before class starts.